scripts/lint-packages.sh

```diff
@@ -127,6 +127,25 @@ lint_package() {
 	echo "Package: $package_name"
 	echo
 
+	echo -n "Layout: "
+	local channel in_dir=''
+	for channel in 'packages' 'x11-packages' 'root-packages'; do
+		[[ -d "$TERMUX_SCRIPTDIR/$channel/$package_name" ]] && {
+			in_dir="$TERMUX_SCRIPTDIR/$channel/$package_name"
+			break
+		}
+	done
+	(( ! ${#in_dir}  )) && {
+		echo "FAIL - '$package_script' is not a directory"
```

Comment:
I suppose this error message is wrong? Should be package_name or in_dir?

Reply:
Keep in mind, if we're in that branch, `$in_dir` is empty since it did not find the package's directory.

Reply:
I think `is not found` or `does not exist` is a better error message.
This feels like is checking build.sh is a folder or not...